Sources/ArrayExtensions.swift

```diff
@@ -132,13 +132,15 @@ extension Array where Element: Equatable {
         self.remove(at: index)
     }
 
-    // EZSE: Removes all occurrences of the given object
+    /// EZSE: Removes all occurrences of the given object(s)
     public mutating func removeAll(_ elements: Element...) {
-        for element in elements {
```

Comment:
(This body refactoring is already present in open PR #412, included here to ease possible merge given that both PR are accepted / reviewed and revised)